src/lib/crypto/krb/t_fortuna.c

```diff
@@ -102,7 +102,7 @@ head_tail_test(struct fortuna_state *st)
     }
 
     for (bit = 0; bit < 8; bit++) {
-        res = ((double)abs(len - bits[bit] * 2)) / (double)len;
+        res = ((double)(len - bits[bit] * 2)) / (double)len;
```

Comment:
I think the code is broken here and should be fixed rather than papered over.  I'm not sure why the test isn't failing; you'd expect at least one of the eight bits to be biased slightly high such that len - bits[bit] * 2 is a huge number.